src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.html

```diff
@@ -99,36 +99,40 @@ <h6>{{ 'ubs-employee.required-fields' | translate }}</h6>
                     <th class="col-region">{{ 'ubs-employee.tariffs-columns.region' | translate }}</th>
                     <th class="col-city">{{ 'ubs-employee.tariffs-columns.city' | translate }}</th>
                     <th class="col-courier">{{ 'ubs-employee.tariffs-columns.courier' | translate }}</th>
-                    <th class="col-actions">{{ 'ubs-employee.tariffs-columns.chat' | translate }}</th>
                     <th class="col-actions">{{ 'ubs-employee.tariffs-columns.tariff' | translate }}</th>
+                    <th class="col-actions">{{ 'ubs-employee.tariffs-columns.chat' | translate }}</th>
                   </tr>
                 </thead>
                 <tbody>
                   <tr *ngFor="let tariff of filteredTariffs">
                     <td>
-                      {{ tariff.region.en }}
+                      {{ langService.getLangValue(tariff.region.ua, tariff.region.en) }}
```

Comment:
Parameter 'filteredTariffs' implicitly has an 'any' type, but a better type may be inferred from usage.
you can use `TariffForEmployee[]`

Also look on 29 line: Type { mask: string } is not assignable to type FactoryArg | undefined...  Assigned type doesn't contain call signatures